boards/cogip2019-cortex/Makefile.include

```diff
@@ -1,6 +1,8 @@
 # define the cpu used by the cogip2018-cortex board
 export CPU = stm32f4
 export CPU_MODEL = stm32f446re
+export CPU_FLASH_SIZE = $(shell expr 512 \* 1024)
```

Comment:
This should be setup into Makefile.include in mcu-firmware base folder.
Some variables already exist from cpu/stm32_common/stm32_mem_lengths.mk and example of use can be found into cpu/stm32_common/Makefile.include.

- https://github.com/RIOT-OS/RIOT/blob/master/cpu/stm32_common/stm32_mem_lengths.mk
- https://github.com/RIOT-OS/RIOT/blob/a01e70a4ab0050331b36ad59946637c62c48ed3a/cpu/stm32_common/Makefile.include#L22

It will make code generic and will lighten platform Makefile

Reply:
Was not aware, i'll take a look . thx